app/controllers/forum/categories/category/threads/thread/posts/index.js

```diff
@@ -42,7 +41,6 @@ export default Controller.extend({
     },
     goToLastPageAndScrollDown() {
       this.set('page', this.totalPages);
-      $('#newForumPost')[0].scrollIntoView(true);
```

Comment:
Also, scrollIntoView is a native javascript function. So you can just use `document.getElementById('newForumPost').scrollIntoView(true)` here.

Reply:
True, but the scroll down is actually more irritating that useful in my opinion